examples/e17_message_components/src/main.rs

```diff
@@ -180,14 +180,17 @@ impl EventHandler for Handler {
             .unwrap();
 
         // Wait for the user to make a selection
-        let mci =
-            match m.await_component_interaction(&ctx).timeout(Duration::from_secs(60 * 3)).await {
-                Some(ci) => ci,
-                None => {
-                    m.reply(&ctx, "Timed out").await.unwrap();
-                    return;
-                },
-            };
+        let mci = match m
+            .await_component_interaction(ctx.shard.clone())
+            .timeout(Duration::from_secs(60 * 3))
```

Comment:
Why cant it be just 180? Isn't it just one more calculation for the machine?
If the reason is to make clear how many minutes it is, couldn't it have a comment?
```suggestion
            .timeout(Duration::from_secs(180)) // 3 minutes
```
(Yes i saw it repeating below vvvv)

Reply:
This shows as changed in my PR due to formatting, I didn't change this.

Reply:
I know, i just noticed that.